ui/app/templates/components/job-row.hbs

```diff
@@ -30,6 +30,9 @@
     {{this.job.namespace.name}}
   </td>
 {{/if}}
+<td data-test-job-nodepool>
+  {{this.job.nodePool}}
+</td>
```

Comment:
This are getting a bit crowded when displaying child jobs 🤔 

<img width="1226" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/63da90a2-7834-4caf-806b-8162fcb6c799">

Perhaps we could omit node pool and namespace if the context is a child job since that information is already in the parent bar?
<img width="1240" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/c3ddc3ee-3f0e-45b4-96b3-61fed4daef5c">

Reply:
Fixed in https://github.com/hashicorp/nomad/pull/17640